tests/test_cli/test_generate_key.py

```diff
@@ -236,3 +243,95 @@ class TestGenerateKeyWithAddKeyWithConnection(
 
     keys_config_path = "agent.connection_private_key_paths"
     args = ["--connection"]  # type: ignore
+
+
+class TestGenerateN(AEATestCaseEmpty):
+    """Test generate N keys."""
+
+    n = 5
+
+    @pytest.mark.parametrize(
+        argnames="crypto", argvalues=(EthereumCrypto, FetchAICrypto)
+    )
```

Comment:
why `FetchAICrypto` and not `CosmosCrypto`?

Reply:
:+1:

Reply:
`FetchAICrypto` is derived from `CosmosCrypto` So I think we can use either one of them

Reply:
We should make all three available

Reply:
[2c7e674](https://github.com/valory-xyz/open-aea/pull/461/commits/2c7e674532f41643b1903b69d2b421a85d7936ed)